### PR TITLE
KO translation for the 3rd page of "Create a visual AMP story" tutorial.

### PR DESCRIPTION
**Pull request by captain-lee**

### CODE REVIEW

**Comment by captain-lee** [comment]
@techhtml, please take a look at this PR.

File: content/docs/design/visual_story/parts_of_story@ko.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+AMP 스토리를 구성하는 부품을 기술용어로는 AMP 컴포넌트라고 표현하는데 AMP
+스토리는 `amp-story`, 페이지는 `amp-story-page` 그리고 레이어는
+`amp-story-grid-layer` 컴포넌트로 표현합니다.
+
+{{ image('/static/img/docs/amp-story-tag-hierarchy.png', 557, 355, align='center seventyfive' ) }}
+
+자, 그럼 `amp-story` 컨테이너를 가지고 본격적으로 이야기를 시작해보겠습니다.
+
+<div class="prev-next-buttons">
+  <a class="button prev-button" href="/docs/design/visual_story/setting_up.html"><span class="arrow-prev">이전</span></a>
```
Comment (bpaduch) [inline content/docs/design/visual_story/parts_of_story@ko.md]: For any links to docs pages, you need to prefix the link with the locale, like "/ko/docs/..."

File: content/docs/design/visual_story/parts_of_story@ko.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  <a class="button next-button" href="/docs/design/visual_story/start_story.html"><span class="arrow-next">다음</span></a>
```
Comment (bpaduch) [inline content/docs/design/visual_story/parts_of_story@ko.md]: Please fix this link as well.

File: content/docs/design/visual_story/parts_of_story@ko.md (hunk 1 above)
Comment (captain-lee) [reply content/docs/design/visual_story/parts_of_story@ko.md]: Thanks, fixed!

File: content/docs/design/visual_story/parts_of_story@ko.md (hunk 2 above)
Comment (captain-lee) [reply content/docs/design/visual_story/parts_of_story@ko.md]: Updated.

**Comment by captain-lee** [comment]
Thanks, @bpaduch. Please take another look.

**Pull request merged**